src/cow-react/modules/limitOrders/containers/OrdersWidget/index.tsx

```diff
@@ -59,11 +60,14 @@ export function OrdersWidget() {
   }, []) // eslint-disable-line react-hooks/exhaustive-deps
 
   return (
-    <Orders
-      tabs={tabs}
-      orders={orders}
-      balancesAndAllowances={pendingBalancesAndAllowances}
-      isWalletConnected={!!account}
-    />
+    <>
+      <Orders
+        tabs={tabs}
+        orders={orders}
+        balancesAndAllowances={pendingBalancesAndAllowances}
+        isWalletConnected={!!account}
+      />
+      <LimitOrdersReceiptModal />
```

Comment:
Would be awesome to open this model using router:
`/#/5/limit-orders/WETH?tab=history&orderId=0xd2b819`
When there is an `orderId` paramer - open modal

Reply:
If it's not so easy to implement now, you can just add TODO for the future

Reply:
Added todo :+1: